database/seeders/AirlineSeeder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Airline;
+use Illuminate\Database\Seeder;
+
+class AirlineSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        Airline::factory()->count(15)->create();
```

Comment:
```suggestion
        AirlineFactory::new()->count(15)->create();
```
Remember that we should interact directly with the factory class, not via a method in the model.